ci-operator/config/openshift/microshift/openshift-microshift-main.yaml

```diff
@@ -72,18 +72,18 @@ tests:
   commands: RPM_INFO=true make rpm srpm
   container:
     from: microshift-test-rpm
-  run_if_changed: ^(rpm/*|packaging/.*|Makefile)
+  run_if_changed: ^(rpm/.*|packaging/.*|Makefile)$
 - as: test-rebase
   cluster: build02
   optional: true
-  run_if_changed: ^(assets/.*|scripts/auto-rebase/.*|go\.mod)
+  run_if_changed: ^(assets/.*|scripts/auto-rebase/.*|go\.mod)$
   steps:
     env:
       DRY_RUN: "y"
     workflow: openshift-microshift-rebase
 - as: microshift-metal-tests
   cluster: build02
-  run_if_changed: ^(assets/.*|cmd/.*|etcd/.*|pkg/.*|packaging/.*|test/.*|vendor/.*|go\.sum|go\.mod|Makefile|.*\.sh|.*\.ks\.template)
+  run_if_changed: ^(assets/.*|cmd/.*|etcd/.*|pkg/.*|packaging/.*|test/.*|vendor/.*|Makefile.*|scripts/.*|.*\.ks\.template)$
```

Comment:
Do we need the `.ks.template` part? It is covered by `test/.*`, right?

Reply:
Done